compiler/z/codegen/S390Instruction.cpp

```diff
@@ -4065,34 +4061,11 @@ uint8_t *
 TR::S390VRRhInstruction::generateBinaryEncoding()
    {
    // Error Checking
-   TR::Register* v1Reg = getRegisterOperand(1);
-   TR::Register* v2Reg = getRegisterOperand(2);
-   TR_ASSERT(v1Reg != NULL, "VRR-h operand should not be NULL!");
-   TR_ASSERT(v2Reg != NULL, "VRR-h Operand should not be NULL!");
-
-   uint8_t * instructionStart = cg()->getBinaryBufferCursor();
-   uint8_t * cursor = instructionStart;
-   memset((void*)cursor, 0, getEstimatedBinaryLength());
-
-   // Copy binary
-   getOpCode().copyBinaryToBuffer(instructionStart);
-
-   // Masks
-   uint8_t mask3 = getM3();
-   setMaskField(reinterpret_cast<uint32_t *>(cursor), mask3, 1);
-
-   // Operands
-   toRealRegister(v1Reg)->setRegister2Field(reinterpret_cast<uint32_t *>(cursor));
-   toRealRegister(v2Reg)->setRegister3Field(reinterpret_cast<uint32_t *>(cursor));
+   TR_ASSERT(getRegisterOperand(1) != NULL, "VRR-h V1 should not be NULL!");
+   TR_ASSERT(getRegisterOperand(2) != NULL, "VRR-h V2 should not be NULL!");
 
-   // Cursor move
-   // update binary length
-   // update binary length estimate error
-   cursor += getOpCode().getInstructionLength();
-   setBinaryLength(cursor - instructionStart);
-   setBinaryEncoding(instructionStart);
-   cg()->addAccumulatedInstructionLengthError(getEstimatedBinaryLength() - getBinaryLength());
-   return cursor;
+   // Generate Binary Encoding
+   return TR::S390VRRInstruction::generateBinaryEncoding();
```

Comment:
The comment here is not very useful as it says the same thing the subsequent function name does. Similarly in the other evaluators. We can probably just remove the comment.

Reply:
Latest commit removes the comment from all the evaluators.